cmd/kpod/stop.go

```diff
@@ -50,19 +50,18 @@ func stopCmd(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrapf(err, "could not update list of containers")
 	}
-	hadError := false
+	var lastError Error
 	for _, container := range c.Args() {
 		cid, err := server.ContainerStop(container, stopTimeout)
 		if err != nil {
-			hadError = true
-			logrus.Error(err)
+			if lastError != nil {
+				logrus.Error(lastError)
+			}
+			lastError = errWrap(err, "failed to stop %v", container)
 		} else {
 			fmt.Println(cid)
 		}
 	}
 
-	if hadError {
-		os.Exit(1)
-	}
-	return nil
+	return lastError
```

Comment:
I said this on the review for kill, but I'm in favor of keeping the existing code structure, but returning a generic "error stopping some containers" error instead of os.Exit(-1)

Reply:
Nevermind, I think you convinced me.